src/disco/css/InstallButton.scss

```diff
@@ -190,3 +170,19 @@ $installStripeColor2: #00C42E;
     }
   }
 }
+
+.progress {
+  background: transparentize(#3ef827, 0.50);
+  bottom: 0;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 0;
+}
+
+@for $i from 1 through 100 {
+  [data-download-progress="#{$i}"] .progress {
+    width: #{percentage($i) / 100};
```

Comment:
There also appears to be a green border on the switch while it is downloading.

![screenshot 2016-05-06 10 05 42](https://cloud.githubusercontent.com/assets/211578/15077038/2a1318ba-1372-11e6-84c8-4dca91e8c8a3.png)